lib/chat/db/scope/invitations_handshaker.ex

```diff
@@ -0,0 +1,265 @@
+defmodule Chat.Db.Scope.InvitationsHandshaker do
+  @moduledoc """
+  Process cargo room invitations by handshaker
+  """
+
+  import Chat.Db.Scope.Utils
+  alias Chat.Dialogs.Dialog
+  alias Chat.Dialogs.Message
+
+  def build_initial_context(snap) do
+    %{}
+    |> put_room_invitations_content(snap, nil)
+    |> put_invitations_keymap()
+    |> put_invitations_messages(snap)
+    |> clear_context_preparation()
+  end
+
+  def put_room_invitations_content(context, snap, keys) do
```

Comment:
all the function in this module are `def`s  
Can we make `defp` for internal ones

Reply:
right, fixed